### DISCUSSION TITLE
Desktop open .zip themes

### DISCUSSION THREAD

**Discussion by ppete2** [devemux86/cruiser#241] [Q&A] [answered]
More and more designfiles of mapsforge maps are offered as .zip file, which are prefered on phones in opposite to their unzipped versions.
Please make it possible to conviniently open these .zip files within Cruiser without the need to unzip them first.

**Comment by devemux86** [comment, marked as answer]
@ppete2 thanks for the suggestion.

Map themes are always offered as zip, as they also contain other files, like symbol images.
The use of zip on Android was forced by the storage restrictions on latest Android versions.

On desktop we do not have such restrictions. The fastest and recommended way is with plain files.
Also it is easier to work in map themes: modify directly the theme files and reload the map with F5.

**Reply by devemux86** [reply]
By the way the maps are also offered in zip by several providers, like [OpenAndroMaps](https://www.openandromaps.org/en) or [Freizeitkarte](https://www.freizeitkarte-osm.de/android/en/index.html).

At the end we have to unzip all downloads, either maps or themes. :slightly_smiling_face: 

On Android they are handled automatically by the Cruiser download manager.

**Comment by ppete2** [comment]
I see, but also in the view regarding software usibility:
Many phone apps and desktop software are able to trasnsparent work with .zip files. No need for the user to work with file explorers, unpack .zip files etc first. Of course the opening of .zip files shouldn't be an exclusive option - users should still be able to unzip them and select the regarding .xml-file like now - if they want.
So it would be easy going to de-zip them by Cruiser and store the plain files within a temp/user directory by Cruiser on desktops.
